src/resources/emco.lua

```diff
@@ -431,9 +431,6 @@ function EMCO:new(cons, container)
   me.tabs = {}
   me.tabsToBlink = {}
   me.mc = {}
-  self.blinkTimerID = tempTimer(me.blinkTime, function()
-    me:doBlink()
-  end, true)
   me:reset()
```

Comment:
```lua
if me.blink then
  me:enableBlink()
end
```

so it enables the timer only if the feature is enabled in the table. It looks like I flubbed by setting self.blinkTimerID instead of me.blinkTimerID there as it is, so good thing you decided to change it